vignettes/RUVIIIPRPS_TCGA_BrCa_RNAseq.Rmd

```diff
@@ -362,9 +382,14 @@ brca.se.obj <- computePCA(
     remove.na = 'none', 
     save.se.obj = TRUE, 
     verbose = TRUE)
-brca.se.obj@metadata$metric$unstranded$fastPCA$svd$d
 ```
 
+Using the calculated PCs, we then generate scatterplots and boxplots to explore different sources of variations. 
+Indicates a name of the column in the sample annotation of the SummarizedExperiment object.
+If the `variable` argument is a continuous variable, the function creates scatterplots of PCs1-3 vs the selected variable. If `variable` is categorical, then the function generates pairwise scatterplots of PCs1-3 coloured by the selected categorical variable; if `plot.type = "boxplot"` for categorical variables, the function creates boxplots of PCs1-3 across the variable.
+
+**NOTE to improve `plotPCA()`**:
+Currently, when `plot.type = "boxplot"` this does not print the boxplots. Need to be updated s that prints the plot.  
```

Comment:
Need to update the plotPCA() function to print the boxplots.

Reply:
Thanks, I have added a option to plot the box plots.